scripts/apitypings/testdata/genericslice/genericslice.ts

```diff
@@ -1,10 +1,13 @@
+// Code generated by 'guts'. DO NOT EDIT.
+
 // From codersdk/genericslice.go
 export interface Bar {
-	readonly Bar: string;
+    readonly Bar: string;
 }
 
 // From codersdk/genericslice.go
+// biome-ignore lint lint/complexity/noUselessTypeConstraint: golang does 'any' for generics, typescript does not like it
 export interface Foo<R extends any> {
-	readonly Slice: Readonly<Array<R>>;
-	readonly TwoD: Readonly<Array<Readonly<Array<R>>>>;
+    readonly Slice: readonly R[];
+    readonly TwoD: readonly R[][];
```

Comment:
Can't remember if we talked about this being out of scope for this PR, but just in case, this 2D array isn't deeply readonly. It's a readonly array of mutable `R` arrays. The fully safe type would have to be `readonly (readonly R[])[]` (as ugly/clunky as it is)

Reply:
Yea, I need to recursively apply it :cry: